pkg/utils/database/postgres.go

```diff
@@ -161,10 +162,19 @@ func (p Postgres) createDatabase(admin AdminCredentials) error {
 		}
 	}
 
+	err := p.enableMonitoring(admin)
```

Comment:
Shouldn't this if condition be here?
```go
if p.Monitoring {..}
```

It seems to me that the `enableMonitoring` function should always try enabling monitoring and should be called only when it's wanted/

Reply:
make completely sense. updated